components/button/floating-buttons.js

```diff
@@ -147,75 +165,98 @@ class FloatingButtons extends RtlMixin(LitElement) {
 		`;
 	}
 
-	_calcContainerPosition() {
-		this._floating = this._shouldFloat();
-		if (!this._floating || !this.offsetParent) {
-			return;
+	updated(changedProperties) {
+		if (changedProperties.has('alwaysFloat')) {
+			this._recalculateFloating();
 		}
+	}
 
-		const offsetParentBoundingRect = this.offsetParent.getBoundingClientRect();
-		const boundingRect = this.getBoundingClientRect();
+	async _calcContainerPosition() {
```

Comment:
Side-by-side diff might be best for this method. Mostly things just got indented inside the `rAF`.